### PR TITLE
8339 add layer id to cartodb layers in named maps templates

### PR DESCRIPTION
**Pull request by gfiorav**
Fixes part of https://github.com/CartoDB/cartodb/issues/8339

CR please @juanignaciosl 

cc/ @alonsogarciapablo @rochoa @xavijam

### CODE REVIEW

**Comment by gfiorav** [comment]
If someone tells me how to bump Mapconfig to 1.5.1 we're done with #8339

**Comment by gfiorav** [comment]
After speaking to @rochoa the Map Config version stuff is clear and is now being sent (and bumped).

The question about torque layers remains though.

**Comment by juanignaciosl** [comment]
👍

**Comment by alonsogarciapablo** [comment]
@guido thank you!

> Also, what about torque layers? Should the have id as well?

I don't see why not, but @rochoa can confirm this.

File: lib/carto/named_maps/template.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ module Carto
   module NamedMaps
     class Template
       NAMED_MAPS_VERSION = '0.0.1'.freeze
+      MAP_CONFIG_VERSION = '1.5.1'.freeze
```
Comment (rochoa) [inline lib/carto/named_maps/template.rb]: This should be 1.5.0.

Comment (alonsogarciapablo) [inline lib/carto/named_maps/template.rb]: Sorry @gfiorav, my bad. I changed it in the issue description.

**Comment by rochoa** [comment]
That's more on your side, I would add ids if they are useful for you as a client. Have in mind Maps API will pick a layer ID for you if you don't send any.

**Comment by alonsogarciapablo** [comment]
Then I'd say yes, let's add those IDs to Torque layers @gfiorav! Thanks!

**Comment by gfiorav** [comment]
Hey @rochoa @alonsogarciapablo , I've added ids for torque layers as well.

Please CR, @juanignaciosl !

File: lib/carto/named_maps/template.rb (hunk 1 above)
Comment (juanignaciosl) [inline lib/carto/named_maps/template.rb]: ^

**Comment by juanignaciosl** [comment]
🌴

File: lib/carto/named_maps/template.rb (hunk 1 above)
Comment (gfiorav) [inline lib/carto/named_maps/template.rb]: changed!

**Comment by gfiorav** [comment]
@rochoa @alonsogarciapablo should we organise some staging deploy? Is there something particular to test? If not this is deploy ready.

**Comment by alonsogarciapablo** [comment]
@gfiorav, If you want to test this...

1. Open the inspector an look for the instantiation request to the Maps API (url should be something like: http://cdb.localhost.lan:8181/api/v1/map/named/tpl_31422150_3797_11e6_a0d1_080027880ca6?stat_tag=31422150-3797-11e6-a0d1-080027880ca6).

2. The response should look something like:
  ```
  {
    "layergroupid": "cdb@da78e72c@51ce037dfcde5516c2a05f13ad36db78:1466604167999",
    "metadata": {
      "layers": [
        {
          "type": "http",
          "id": "http-layer0",
          "meta": {
            "stats": []
          }
        },
        {
          "type": "mapnik",
          "id": "THE_LAYER_ID",
          "meta": {
            "stats": [],
            "cartocss": "..."
          }
        }
      ],
      "dataviews": {},
      "analyses": []
   },
    "last_updated": "2016-06-22T14:02:47.999Z"
  }
  ```

3. Verify that the "id" of the "mapnik" layer (whose value is "LAYER_ID" in this case) matches the ID of the layer in the named map template.

Ping me if you have questions. Thanks

**Pull request merged**